src/main/java/javax/validation/ConstraintValidatorContext.java

```diff
@@ -326,7 +330,8 @@ ContainerElementNodeBuilderCustomizableContext addContainerElementNode(String na
 			 *
 			 * @since 2.0
 			 */
-			LeafNodeBuilderCustomizableContext inContainer(Class<?> containerClass, Integer typeArgumentIndex);
+			LeafNodeBuilderCustomizableContext inContainer(Class<?> containerClass,
+														   Integer typeArgumentIndex);
```

Comment:
I agree with you and don't like this style either. But it's what's used in the API in other places AFAICS, so I'd prefer to stick to it as far as possible.